website/docs/r/ovh_dedicated_server_networking.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_dedicated_server_networking"
+sidebar_current: "docs-ovh-resource-ovh-dedicated-server-networking"
+description: |-
+  Manage dedicated server networking interface
+---
+
+# ovh_dedicated_server_networking
+
+Manage dedicated server networking interface.
+
+## Example Usage
+
+The following example aims to bond all interfaces in vRack.
+
+```hcl
+locals {
+  dedicated_server = "nsXXXXXX.ip-XX-XXX-XX.eu"
+}
+
+data "ovh_dedicated_server" "server" {
+  service_name = local.dedicated_server
+}
+
+resource "ovh_dedicated_server_networking" "server"{
+  service_name = local.dedicated_server
+  interfaces {
+      macs = sort(flatten(data.ovh_dedicated_server.server.vnis.*.nics)),
+      type = "vrack"
+  }
+} 
+```
+
+The following example aims to attach the server to two different vRack.
+
+```hcl
+locals {
+  dedicated_server = "nsXXXXXX.ip-XX-XXX-XX.eu"
+}
+
+data "ovh_dedicated_server" "server" {
+  service_name = local.dedicated_server
+}
+
+resource "ovh_dedicated_server_networking" "server"{
+  service_name = local.dedicated_server
+  # Bond the two first interfaces in the first vrack
+  interfaces {
+      macs = slice(sort(flatten(data.ovh_dedicated_server.server.vnis.*.nics)), 0, 2),
+      type = "vrack"
+  }
+  # Bond the two last interfaces in the last vrack
+  interfaces {
+      macs = slice(sort(flatten(data.ovh_dedicated_server.server.vnis.*.nics)), 2, 4),
+      type = "vrack"
+  }
+} 
+```
+
+## Arguments Reference
+
+The following arguments are required:
+
+* `service_name` - (String) The service_name of your dedicated server.
+* `interfaces` - (Block List, Min: 1, Max: 2) Interface or interfaces aggregation.
+
+### Nested Schema for interfaces
+
+Required:
+
+* `macs` (List of String) List of mac addresses to bond together.
+* `type` (String) Type of bonding to create.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `description` - Operation description.
+* `status` - status of the networking configuration (should be `active`).
```

Comment:
If importer have been implemented, please add it in the documentation also.
Example:
https://registry.terraform.io/providers/ovh/ovh/latest/docs/resources/cloud_project_kube#import